contracts/governance/Governor.sol

```diff
@@ -117,17 +117,15 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
      * can be produced from the proposal data which is part of the {ProposalCreated} event. It can even be computed in
      * advance, before the proposal is submitted.
      *
-     * Note that the chainId and the governor address are not part of the proposal id computation. Consequently, the
-     * same proposal (with same operation and same description) will have the same id if submitted on multiple governors
-     * across multiple networks. This also means that in order to execute the same operation twice (on the same
-     * governor) the proposer will have to change the description in order to avoid proposal id conflicts.
+     * Note that the chainId and the governor address are not part of the proposal id computation, but can be added to
+     * guarantee more uniqueness if the same proposal is to submitted on multiple governors across multiple networks.
```

Comment:
I feel some context in the previous comment was relevant, let's do:

```suggestion
     * Note that the chainId and the governor address are not part of the proposal id computation. Consequently, the
     * same proposal will share its id with another governor in a different network where the same proposal was
     * submitted. We recommend using the description field as a discriminator to avoid proposal id conflicts.
     * Similarly, developers can add both `chainId` and `address(this)` to the hash computation by overriding
     * this function.
     *
     * IMPORTANT: Overriding this function may have unexpected effects after an upgrade. Consider to verify
     * that the new implementation doesn't allow a proposer to forge an id.
```